conanfile.py

```diff
@@ -76,51 +77,52 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
-        if not self.options.namespace_begin:
-            self.options.namespace_begin = (
-                f'namespace {self.options.namespace} {{'
-            )
-        if not self.options.namespace_end:
-            self.options.namespace_end = '}'
-
     def layout(self):
         cmake_layout(self)
 
     def requirements(self):
-        self.requires('boost/1.79.0')
-        self.requires('c-ares/1.18.1')
-        self.requires('cctz/2.3')
-        self.requires('concurrentqueue/1.0.3')
+        self.requires('boost/1.79.0', transitive_headers=True)
+        self.requires('c-ares/1.19.1')
+        self.requires('cctz/2.3', transitive_headers=True)
+        self.requires('concurrentqueue/1.0.3', transitive_headers=True)
         self.requires('cryptopp/8.7.0')
-        self.requires('fmt/8.1.1')
+        self.requires('fmt/8.1.1', transitive_headers=True)
         self.requires('libnghttp2/1.51.0')
         self.requires('libcurl/7.86.0')
         self.requires('libev/4.33')
         self.requires('http_parser/2.9.4')
         self.requires('openssl/1.1.1s')
-        self.requires('rapidjson/cci.20220822')
+        self.requires('rapidjson/cci.20220822', transitive_headers=True)
         self.requires('yaml-cpp/0.7.0')
         self.requires('zlib/1.2.13')
 
         if self.options.with_jemalloc:
-            self.requires('jemalloc/5.2.1')
+            self.requires('jemalloc/5.3.0')
         if self.options.with_grpc:
-            self.requires('grpc/1.48.0')
+            self.requires('grpc/1.48.4', transitive_headers=True, transitive_libs=True)
```

Comment:
Conan changed grpc recipe. Previously googleapis and grpc-proto were a part of grpc package. But then they unvendored them to separate packages.